azure/posix.yml

```
@@ -63,6 +63,7 @@ jobs:
       - bash: |
           set -e
           pip install virtualenv wheel
+          pip install versioneer[toml]
           BUILD_DEPENDS="$NP_BUILD_DEP $CYTHON_BUILD_DEP"
```

Maybe this:

```suggestion
          BUILD_DEPENDS="$NP_BUILD_DEP $CYTHON_BUILD_DEP versioneer[toml]"
```

Feels like the missing dependency is inside the docker image, and you're installing it in the CI image above.